content/docs/cucumber/api.md

```diff
@@ -50,6 +50,10 @@ In this case, the `DataTable` is automatically flattened to a `List<String>`
 by Cucumber (using `DataTable.asList(String.class)`) before invoking the step definition.
 {{% /text %}}
 
+{{% text "javascript" %}}
```

Comment:
Thanks for adding the link; please add even a brief description of what this will link to. Something like @luke-hill suggested is fine.